variables.md

```diff
@@ -13,6 +13,7 @@ ADDONURL_*      | string    |               | Define url for the addons list def
 ASSERT_Y2LOGS   | boolean   | false         | If set to true, we will parse YaST logs after installation and fail test suite in case unknown errors were detected.
 AUTOCONF        | boolean   | false         | Toggle automatic configuration
 AUTOYAST        | string    |               | Full url to the AY profile or relative path if in [data directory of os-autoinst-distri-opensuse repo](https://github.com/os-autoinst/os-autoinst-distri-opensuse/tree/master/data). If value starts with `aytests/`, these profiles are provided by suport server, source code is available in [aytests repo](https://github.com/yast/aytests-tests)
+AUTOYAST_PRE_ENCRYPT_DISK | string | | Disk to be pre-partioned before AutoYaST installation which will contain one encrypted partition, i.e.: vdb
```

Comment:
Can you add the ENCRYPT_ACTIVATE_EXISTING?

Reply:
existing code.

Reply:
ENCRYPT_ACTIVATE_EXISTING is not described in the file